doc/advanced-configuration/outgoing-connections.md

```diff
@@ -54,6 +54,38 @@ will be expanded to the following configuration:
 {redis, <<"d.eu">>, default, PoolOpts, ConnOpts}
 ```
 
+## HTTP connections setup
+
+Some MongooseIM modules needs HTTP connection to external service.
+These pools needs to be configured and started before the module needs them.
+Sample configuration looks like the following:
+
+```erlang
+{outgoing_pools, [
+ {http, global, default, PoolOptions, ConnectionOptions}
+]}.
+```
+
+`PoolOptions` are described above, below there are described recommended `PoolOptions` for `HTTP` pools:
```

Comment:
PoolOptions` are described above, below there are the recommended `PoolOptions` for `HTTP` pools: